go.mod

```diff
@@ -9,7 +9,6 @@ require (
 	github.com/sirupsen/logrus v1.8.1
 	github.com/spf13/cobra v1.1.3
 	google.golang.org/grpc v1.37.0
-	google.golang.org/grpc/cmd/protoc-gen-go-grpc v1.1.0 // indirect
```

Comment:
Running `bin/gen-proto` brings these lines back, but running `go mod tidy` deletes them.  I think the consensus is that they should be left in.

Reply:
Actually, I think this can be fixed by adding a `tools.go`:

```console
> cat tools.go
// +build tools

package main

import (
	_ "google.golang.org/grpc/cmd/protoc-gen-go-grpc"
)
```